AzureCalling/Controllers/CallViewController.swift

```diff
@@ -510,6 +529,18 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
         meetingInfoViewUpdate()
     }
 
+    @objc func onRemoteParticipantIsMutedChanged(_ notification: Notification) {
```

Comment:
My concern here is if pass RemoteParticipant through notification is sensible. 
Usually we don't pass an object in the notification, instead we could do `delegate`, `closure`.
My original thought is 1. check if the remoteParticipant is in the list 2.0 If no, no notification post. 2.1 If yes, post notification 3. All participant view will get updates. 
Please chat with Pavel to sync the thought here.

Reply:
I've tried both targeted updates and update all approach. The former seems instantaneous whereas the latter can have some delay in the updates. Somehow we do need to pass data for the next task when we want to identify active speaker.

So I've looked through stackoverflow and some blogs. It's fine to use notification to pass data, especially useful when it's broadcasting from one class to multiple classes, although we need to pass in the correct arguments to [post(name:object:userInfo:)](https://developer.apple.com/documentation/foundation/notificationcenter/1410608-post) for it to be semantically correct. `object` applies to the sender of the notification, whereas `userInfo` is used to pass additional data which we can utilize.

Reply:
I think we should update all for now, and optimize targeting specific participant views separately. Unless there is something causing extra time for each participant view update, there should not be a noticeable delay since we are updating 4 max.

Reply:
Changed to update just the mute indicator for all remote participants if there're any mute toggling events.

Reply:
What I meant was we should just be able to call `queueParticipantViewsUpdate()`.

Reply:
Changed to use `queueParticipantViewsUpdate()`, though there's a 1-2 seconds delay from my experience